docs/Developers/Writing_Linter_Bears.rst

```diff
@@ -71,7 +71,8 @@ appropriate parameters. This is done inside ``create_arguments``,
 - ``file``: The contents of the file to process, given as a list of lines
   (including the return character).
 - ``config_file``: The absolute path to a config file to use. If no config file
-  is used, this parameter is ``None``. More on that later.
+  is used, this parameter is ``None``. Processing of the config file
+  is left to the Bear's implementation of the method.
```

Comment:
So it is just a file I can give a bear right? Everything else is implemented in the bear?